app/services/reports/acquisition_report_service.rb

```diff
@@ -0,0 +1,120 @@
+module Reports
+  class AcquisitionReportService
+    include ActionView::Helpers::NumberHelper
+    attr_reader :year, :organization
+
+    def initialize(year:, organization:)
+      @year = year
+      @organization = organization
+    end
+
+    def report
+      @report ||= Reports::Report.new('Diaper Acquisition', [
+                                        { 'Disposable diapers distributed' => number_with_delimiter(distributed_diapers) },
+                                        { 'Average monthly disposable diapers distributed' => number_with_delimiter(monthly_disposable_diapers) },
+                                        { 'Total diaper drives' => annual_drives.count },
+                                        { 'Disposable diapers collected from drives' => number_with_delimiter(disposable_diapers_from_drives) },
+                                        { 'Money raised from diaper drives' => number_to_currency(money_from_drives) },
+                                        { 'Total diaper drives (virtual)' => virtual_diaper_drives.count },
+                                        { 'Money raised from diaper drives (virtual)' => number_to_currency(money_from_virtual_drives) },
+                                        { 'Disposable diapers collected from drives (virtual)' => number_with_delimiter(disposable_diapers_from_virtual_drives) },
+                                        { '% diapers donated' => percent_donated.round },
+                                        { '% diapers bought' => percent_bought.round },
+                                        { 'Money spent purchasing diapers' => number_to_currency(money_spent_on_diapers) },
+                                        { 'Purchased from' => purchased_from },
+                                        { 'Vendors diapers purchased through' => vendors_purchased_from }
+                                      ])
+    end
+
+    def distributed_diapers
+      @distributed_diapers ||= organization
+                               .distributions
+                               .for_year(year)
+                               .joins(line_items: :item)
+                               .merge(Item.disposable)
+                               .sum('line_items.quantity')
+    end
+
+    def monthly_disposable_diapers
+      (distributed_diapers / 12.0).round
+    end
+
+    def annual_drives
+      organization.diaper_drives.within_date_range("#{year}-01-01 - #{year}-12-31")
+    end
+
+    def disposable_diapers_from_drives
+      annual_drives.joins(donations: { line_items: :item }).merge(Item.disposable).sum(:quantity)
+    end
+
+    def money_from_drives
+      annual_drives.joins(:donations).sum(:money_raised) / 100
+    end
+
+    def virtual_diaper_drives
+      annual_drives.where(virtual: true)
+    end
+
+    def money_from_virtual_drives
+      virtual_diaper_drives.joins(:donations).sum(:money_raised) / 100
```

Comment:
yesssss -- this is good. I think it might be better to do it this way rather than risk an accidental N+1 via the model method